doc/md/cli.md

```diff
@@ -0,0 +1,160 @@
+---
+title: CLI Reference
+id: cli-reference
+slug: /cli/reference
+---
+## Introduction
+
+This document serves as reference documentation for all available commands in the Atlas CLI.
+Similar information can be obtained by running any atlas command with the `-h` or `--help`
+flags.
+
+For a more detailed introduction to the CLI capabilities, head over to the
+[Getting Started](/cli/getting-started/setting-up) page.
+
+
+## atlas env
+
+Print atlas env params
+
+
+Env prints atlas environment information.
+Every set environment param will print in the form of NAME=VALUE.
+
+List of supported environment parameters:
```

Comment:
This doesn't look good in the rendered document, maybe add `\n`
![image](https://user-images.githubusercontent.com/5472778/149977447-cedf53f6-e836-4635-873b-be03e09408b4.png)